File: example/dead_scheduler.js
const sleep = async (ms) => {
    return new Promise(resolve => setTimeout(resolve, ms))
}

const dead = async () => {
    while (true) {
        // await sleep(1000)
    }
}

function main() {
    dead()

    while (true) {
        console.log('main')        
    }
}

main()

File: example/dead_scheduler.go
package main

import (
	"time"
	"fmt"
)

func dead() {
	i := 0
	for i < 0 {
		i++
	}
}

func main() {
	go dead()

	for {
		fmt.Println("1")
		time.Sleep(time.Second)
	}
}
